solr/core/src/java/org/apache/solr/handler/AnalysisRequestHandlerBase.java

```diff
@@ -82,7 +82,7 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
    *
    * @throws Exception When analysis fails.
    */
-  protected abstract NamedList doAnalysis(SolrQueryRequest req) throws Exception;
+  protected abstract NamedList<?> doAnalysis(SolrQueryRequest req) throws Exception;
```

Comment:
I think best practice is to use `NamedList<Object>` as the return type, and `NamedList<?>` as the argument type in methods, but I can't find a reference for it right now.